### PR TITLE
fix: crop modal

### LINKED ISSUE #128: Uploading avatar does not show cropped image
Screenshot 2024-03-05 at 09 41 47

However it works in builder for item thumbnails.

### PR DESCRIPTION
**Pull request by spaenleh**
In this PR:
- fix #128 
- move pages to a `pages` directory
- uniformise the page title appearences

### CODE REVIEW

File: src/config/constants.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export const THUMBNAIL_EXTENSION = 'image/jpeg';
 export const THUMBNAIL_SETTING_MAX_WIDTH = 200;
 export const THUMBNAIL_SETTING_MAX_HEIGHT = 200;
 
-export const AVATAR_ICON_HEIGHT = 30;
+export const AVATAR_SIZE = 256;
```
Comment (pyphilia) [inline src/config/constants.ts:52]: Is it related to some thumbnail size?

File: src/pages/AvatarSettingsScreen.tsx [hunk 2]
```diff
@@ -111,37 +117,43 @@ const AvatarSettings = (): JSX.Element | null => {
...
+          {t('PROFILE_AVATAR_TITLE')}
+        </Typography>
+        <Typography variant="body1">
+          {t('PROFILE_AVATAR_INFORMATION')}
+        </Typography>
+        <input
+          type="file"
+          accept="image/*"
+          onInput={onSelectFile}
+          // onChange is successfully triggered in test
```
Comment (pyphilia) [inline src/pages/AvatarSettingsScreen.tsx:131]: Remove?

File: src/pages/MemberScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+              </Grid>
+            </Grid>
+            <Grid container alignItems="center">
+              <Grid item xs={4}>
+                <Typography>{t('PROFILE_LANGUAGE_TITLE')}</Typography>
+              </Grid>
+              <Grid item xs={8}>
+                <LanguageSwitch
+                  memberId={member.id}
+                  lang={(member.extra?.lang as string) || DEFAULT_LANG}
```
Comment (pyphilia) [inline src/pages/MemberScreen.tsx:105]: Does it work?
```suggestion
                  lang={member.extra?.lang ?? DEFAULT_LANG}
```

**Review by pyphilia — APPROVED** [review]
🐑

File: src/config/constants.ts (hunk 1 above)
Comment (spaenleh) [reply src/config/constants.ts:52]: Yes the medium is 256px wide.

**Pull request merged**